docs/reference/networks.mdx

```diff
@@ -6,7 +6,7 @@ description: Specs for Stellar's various networks
 
 :::info
 
-To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/).
+To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/) and the [Stellar Status](https://status.stellar.org/) for network and service uptime.
```

Comment:
```suggestion
To see the current status of Mainnet and Testnet, please see the [Stellar.org Dashboard](https://dashboard.stellar.org/) and the [Stellar Statuspage](https://status.stellar.org/) for network and service uptime.
```

Reply:
I think statuspage is two words.

Reply:
I thought so, too. But the atlassian product that is running the status website calls itself "statuspage" 🤷‍♂️